trinity/components/builtin/attach/console.py

```diff
@@ -91,11 +93,12 @@ def console(ipc_path: Path,
 
     # wait to import web3, because it's somewhat large, and not usually used
     import web3
-    ipc_provider = web3.IPCProvider(ipc_path)
+    # Type bug: https://github.com/ethereum/web3.py/issues/1647
+    ipc_provider = web3.IPCProvider(ipc_path)  # type: ignore
     w3 = web3.Web3(ipc_provider)
 
     # Allow omitting params by defaulting to `None`
-    def rpc(method: str, params: Dict[str, Any] = None) -> str:
+    def rpc(method: RPCEndpoint, params: Dict[str, Any] = None) -> RPCResponse:
         return ipc_provider.make_request(method, params)
 
     namespace = merge({'w3': w3, 'rpc': rpc}, env)
```

Comment:
Maybe I should just change to:
```suggestion
    namespace = merge({'w3': w3, 'rpc': ipc_provider.make_request}, env)
```